ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/jsonrpc/response/JsonRpcError.java

```diff
@@ -27,6 +27,12 @@
   INTERNAL_ERROR(-32603, "Internal error"),
   METHOD_NOT_ENABLED(-32604, "Method not enabled"),
 
+  // eth_sendTransaction specific error message
+  ETH_SEND_TX_NOT_AVAILABLE(
+      -32604,
+      "The method eth_sendTransaction is not supported. Use eth_sendRawTransaction to send a signed transaction to Besu."),
```

Comment:
nit: all other messages doesn't have a full stop. Also, isn't ethsigner client agnostic? i.e. should we use web3 provider (or some other noun instead of Besu?)

Reply:
This is copy from Besu, rather it just matched what Besu does even if it is grammatically inconsistent

Reply:
👍